help/actions/_posts/2015-02-09-export.md

```diff
@@ -6,7 +6,51 @@ order: 3
 
 ### Export Formats
 
-Use this button to export your search results in different formats.  Currently available options are ADS Classic, BibTeX, AASTeX and Endnote.
+Use this button to export your search results in different formats.  Currently available options are:
```

Comment:
It's not clear what all of the options in the three lists below correspond to in the export format dropdown menu in BBB. For the following formats, can you add in the menu option text, maybe in parentheses at the end of the description, or at the beginning of the description with a colon following, or something?
* generic tagged abstracts
* Refman format
* SoPh format
* Dublin Core XML
* XML
* XML with abstracts

The other options are pretty self-explanatory, so up to you if you want to add the menu text or not to the lists

Reply:
matched everything.